### PR TITLE
return value of address in ImportMultiSigContractAddr()

### PR DESCRIPTION
**Pull request by HandsomeJeff**
**What current issue(s) does this address, or what feature is it adding?**
Currently we are unable to directly get the address of a multisig contract. This is for ease-of-use for creating custom python programs. 

**How did you solve this problem?**
I added `return address`.

**How did you make sure your solution works?**
It's a simple, quality of life improvement

**Are there any special changes in the code that we should be aware of?**
Just two words

**Please check the following, if applicable:**

- [x ] Did you add any tests?
- [x ] Did you run `make lint`?
- [x ] Did you run `make test`?
- [x ] Are you making a PR to a feature branch or development rather than master?
- [ ] Did you add an entry to `CHANGELOG.rst`? (if not, please do)

### CODE REVIEW

File: neo/Prompt/Commands/LoadSmartContract.py [hunk 1]
```diff
@@ -235,5 +235,7 @@ def ImportMultiSigContractAddr(wallet, args):
         wallet.AddContract(verification_contract)
 
         print("Added multi-sig contract address %s to wallet" % address)
+        
+        return address
 
     return 'Hello'
```
Comment (metachris) [inline neo/Prompt/Commands/LoadSmartContract.py]: Might this `return 'Hello'` not be a problem for your use-case?

**Comment by metachris** [comment]
Looks good. Please also add an entry to the changelog file

File: neo/Prompt/Commands/LoadSmartContract.py (hunk 1 above)
Comment (HandsomeJeff) [reply neo/Prompt/Commands/LoadSmartContract.py]: In my use-case, I'll be interacting with the newly created multisig contract address. So having ImportMultiSigContractAddr() return this address would be helpful.

**Comment by localhuman** [comment]
looks like a codestyle issue is failing the build

**Comment by HandsomeJeff** [comment]
Hi, is there anything I can do on my end?

**Comment by metachris** [comment]
Yes, please run `make lint` and fix the issue.

Also please add a CHANGELOG entry to this PR.

**Comment by localhuman** [comment]
Thanks!

**Pull request merged**